### ISSUE TITLE
`valuesToStrings` is ignored when using the static `format()` and `unformat()` functions

### ISSUE THREAD

**Issue by AlexandreBonneau** [autoNumeric/autoNumeric#489]
```js
const theValue = 0;
AutoNumeric.format(theValue, [
    'percentageEU2dec',
    {
        decimalPlaces  : 0,
        valuesToStrings: {
            0: '-',
        },
    },
]);
```
should return `'-'` when `theValue` is equal to `0`, not `0 %`.

**Comment by AlexandreBonneau** [comment]
The `unformat()` function should also use the `valuesToStrings` option:
```js
AutoNumeric.unformat('-', [
    'euro',
    { valuesToStrings: AutoNumeric.options.valuesToStrings.zeroDash }
]); // '0'
```

**Comment by AlexandreBonneau** [comment]
Fixed in `v4.1.0-beta.20`
